### ISSUE TITLE
Number matching is returning a wrong match

### ISSUE THREAD

**Issue by renatofilho** [googlei18n/libphonenumber#791]
Trying to match: "+28909" with "+13611361530" returns "SHORT_NSN_MATCH";

In my opinion it should return NO_MATCH;

I created a pull request with the a patch to fix the problem.

https://github.com/googlei18n/libphonenumber/pull/790

**Comment by roes** [comment]
Thank you for your report, we will look into it.

**Comment by qiuxinyidian** [comment]
did they fixed？

**Comment by penmetsaa** [comment]
@qiuxinyidian 

No, we didn't get around to fixing this yet. Please see the discussion at https://github.com/googlei18n/libphonenumber/pull/790 for our analysis and a coarse plan for a fix.
